content/en/docs/collector/troubleshooting.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Troubleshooting
+weight: 32
+---
+
+The OpenTelemetry Collector supports a variety of metrics, logs, and extensions
+for troubleshooting collector health and performance.
+
+Detailed recommendations, including common problems, are detailed in the
+OpenTelemetry Collector GitHub repo's
+[troubleshooting document](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md).
```

Comment:
I am wondering if this document should live on opentelemetry.io as well? cc @open-telemetry/collector-maintainers ?

Reply:
Yes! As is already mentioned in the following PR under "Collector":

- #833 

There is valuable information under [opentelemetry-collector/docs](https://github.com/open-telemetry/opentelemetry-collector/tree/main/docs), which (IMHO) would benefit from the increased visibility of being hosted directly on the OTel website.